export default VerifyOtp = async(req, resizeBy, next) =>
{
    try{

    }
    catch(err)
    {
        console.log(err);
        return next("the error is")
    }
}